README.md

```diff
@@ -1,19 +1,36 @@
-# jupyterlab_suggestions
+<h1 align="center">jupyterlab_suggestions</h1>
```

Comment:
Maybe we could also rename the heading to "jupyter-suggestions" to match the name of the repo?

Reply:
Done